generators/app/index.js

```diff
@@ -63,7 +63,7 @@ module.exports = class extends Generator {
 
   writing() {
     this.destinationRoot(this.answers.name);
-
+    
```

Comment:
Removing extra whitespace
```suggestion

```

Reply:
Well, this didn't do what I wanted... was hoping to keep the newline, but without spaces. We'll catch it on the next PR.